frontend/src/screens/researcher/researcher-profile.tsx

```diff
@@ -0,0 +1,217 @@
+                                        <Avatar />
+                                        <ProfileForm className='col-10'/>
+                                    </Box>
+                                </Box>
+                            </ProfileSection>
+
+                            <ProfileSection direction='column' gap='large'>
+                                <h5 className='fw-bolder'>Research Agreements</h5>
+                                <IRB/>
+                                {/*<TermsOfUse/>*/}
+                            </ProfileSection>
+                        </Box>
+                    </Box>
+                </Col>
+
+                <Col sm={3} className='pl-3'>
+                    <Resources direction='column' gap='small'>
+                        <ResourceLinks />
+                        <Box gap='medium' className='mt-1' align='center'>
+                            <CustomerSupportImage height={100} />
+                            <Box direction='column'>
+                                <HelpLink/>
+                            </Box>
+                        </Box>
+                    </Resources>
+                </Col>
+            </Box>
+
+            <Footer className='mt-auto' />
+        </PageWrapper>
+    )
+}
+
+const IRB = () => {
+    return (
+        <Box gap='large' className='container-fluid'>
+            <Col sm={2}>
+                <h6>IRB Detail</h6>
+            </Col>
+            <Col sm={8}>
+                <Box direction='column' gap css={{ border: '1px solid grey', padding: 15, width: 400 }}>
+                    <Box justify='between'>
+                        <img alt="Rice University logo" css={{ width: 120, height: 50 }} src={RiceLogoURL} className='col-6'/>
+                        <Box direction='column' className='col-6 x-small'>
+                            <span>IRB Number: IRB-FY2022-19</span>
+                            <span css={{ color: colors.grayText }}>Expires on 09-01-2026</span>
+                        </Box>
+                    </Box>
+                    <Box direction='column' className='small'>
+                        <Box justify='between'>
+                            <span className='col-6'>Principal Investigator:</span>
+                            <span className='col-6'>Richard G Baraniuk</span>
+                        </Box>
+                        <Box justify='between'>
+                            <div className='col-6'>Institution Name:</div>
+                            <div className='col-6'>Rice University</div>
+                        </Box>
+                    </Box>
+                </Box>
+            </Col>
+            <Col sm={2}>
+                <a href='https://drive.google.com/file/d/1x1M8EcrOOu5U1ZQAtVmhvH3DkTlhtc8I/view' target='_blank'>
+                    <span>Check Details</span>
+                    <Icon icon="right" />
+                </a>
+            </Col>
+        </Box>
```

Comment:
This gets a little ugly on a narrowish (<900px) screen:
<img width="699" alt="image" src="https://user-images.githubusercontent.com/79566/218805171-bc8b69a6-254c-4b94-a8a0-4d73377e9378.png">

I think if you enclose them in `Row` , or just add `.row` to Box they'd wrap like the the Col should.